base/src/com/google/idea/blaze/base/sync/projectstructure/ModuleEditorImpl.java

```diff
@@ -39,11 +39,12 @@
 import com.intellij.openapi.roots.impl.ModifiableModelCommitter;
 import com.intellij.openapi.vfs.VfsUtil;
 import com.intellij.openapi.vfs.VirtualFile;
+
+import javax.annotation.Nullable;
```

Comment:
where is this used?

Reply:
Got reordered , its here https://github.com/bazelbuild/intellij/blob/f17f006ac6c1d94711958038af383ca86a680fba/base/src/com/google/idea/blaze/base/sync/projectstructure/ModuleEditorImpl.java#L114